src/DBFTPlugin/DBFTPlugin.cs

```diff
@@ -87,7 +88,12 @@ public void Start(Wallet wallet)
         private bool RemoteNode_MessageReceived(NeoSystem system, Message message)
         {
             if (message.Command == MessageCommand.Transaction)
-                consensus?.Tell(message.Payload);
+            {
+                Transaction tx = (Transaction)message.Payload;
+                if (tx.SystemFee > settings.MaxBlockSystemFee)
```

Comment:
I have two questions here, 1. we can not access systemfee. 2. why not systemfee+networkfee
```suggestion
                if (tx.SystemFee + tx.NetworkFee > settings.MaxBlockSystemFee)
```